### DISCUSSION TITLE
Versionising

### DISCUSSION THREAD

**Discussion by romanyczn1** [kingslay/KSPlayer#660] [Q&A]
How can i add a specific version of package?

**Comment by kingslay** [comment]
什么意思呢？没有明白

**Reply by romanyczn1** [reply]
you have last tag 2.1.2 on 30 august
I want to have the ability to specify concreate version of your library and dont install updates.

**Comment by kingslay** [comment]
那你直接用spm就可以指定了版本号了

**Reply by romanyczn1** [reply]
Yes, thats what i want, i want you to specify concrete version of your updates

**Comment by yourdearfriend** [comment]
Yes, your latest Version 2.1.2 is older than the currently developed branche. Please,  make a new Build from the current one.

**Comment by yourdearfriend** [comment]
Thanks!
